test/testFiles/simpleProgram.txt

```diff
@@ -0,0 +1,7 @@
+project ZooProject
+modules []
+dir src/AnimalModels
+	interface Animal
+		fields public [string name, number age]
```

Comment:
space required before/after square brackets (otherwise it'll fail once this gets merged to master)